docs/source/howto/ssh.rst

```diff
@@ -0,0 +1,255 @@
+.. _how-to:ssh:
+
+****************************
+How to setup SSH connections
+****************************
+
+AiiDA communicates with remote computers via the SSH protocol.
+There are two ways of setting up an SSH connection for AiiDA:
+
+ 1. Using up a passwordless SSH key (easier, less safe)
+ 2. Using a password-protected SSH key through ``ssh-agent`` (one more step, safer)
+
+.. _how-to:ssh:passwordless:
+
+Using a passwordless SSH key
+============================
+
+
+There are numerous tutorials on the web, see e.g. `here <https://www.redhat.com/sysadmin/passwordless-ssh>`_.
+Very briefly, first create a new private/public keypair (``aiida``/``aiida.pub``), leaving passphrase emtpy:
+
+.. code-block:: console
+
+   $ ssh-keygen -t rsa -f ~/.ssh/aiida
```

Comment:
has it been verified that we really no longer need the `-m PEM` switch? I seem to remember needing this relatively recently

Reply:
As mentioned in the inital Pr comment:

>  I changed  `ssh-keygen -t rsa -b 4096 -m PEM` to `ssh-keygen -t rsa`, since I saw that https://github.com/paramiko/paramiko/issues/340 was closed. But now I notice this was only changed recently: https://github.com/aiidateam/aiida-core/issues/3279, @ltalirz can you confirm if this is still required?

Reply:
I just checked - this was merged in this PR https://github.com/paramiko/paramiko/pull/1343
and should therefore be in paramiko 2.7
We only require paramiko ~=2.6 i.e. this can *not* yet be changed (I will open a PR specifically for this)